### ISSUE TITLE
M68K: attribute suffixes for branches

### ISSUE THREAD

**Issue by maximumspatium** [aquynh/capstone#1668]
Capstone engine uses the .B/.W/.L suffixes for the branch attribute. Examples: 
```
./cstool m68k40 "60 0c"
bra.b	$e

./cstool m68k40 "61 00 00 38"
bsr.w	$3a

./cstool m68k40 "60 FF 00 00 E9 3E"
bra.l	$e940
```
While being consequent, the common suffix for the short branch is `BRA.S` though. That's what the most 68k disassemblers out here will generate for the first instruction.

Question: should we use the more common syntax for short branches?

**Comment by huth** [comment]
I'm also more used to "bra.s" instead of "bra.b" from the Atari assemblers/disasseblers. So FWIW, I'm in favour of switching to .s instead of using .b here.

**Comment by troed** [comment]
A tool I use has recently begun implementing Capstone and this issue has popped up. I second the use of bra.s - I've never seen the other use either on the native platform or with modern cross assemblers.

**Comment by d-hedberg** [comment]
Indeed, bra.s is my preferred choice as well!

**Comment by czietz** [comment]
I have to agree with the previous posters: The mnemonic for a _short_ branch has always been `BRA.S` -- all the way back to the original 68k assembler by Motorola: https://archive.org/details/bitsavers_motorolave000AsmJul83_3899531/page/n15/mode/2up?view=theater
